t/03.scripts/peekJob_output.t

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env perl
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2019] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+use strict;
+use warnings;
+
+use Test::More;
+use Data::Dumper;
+use Test::JSON;
+use JSON qw(decode_json);
+
+use Capture::Tiny ':all';
+use Bio::EnsEMBL::Hive::Utils::Test qw(init_pipeline runWorker beekeeper get_test_url_or_die run_sql_on_db peekJob);
+
+
+# eHive needs this to initialize the pipeline (and run db_cmd.pl)
+$ENV{'EHIVE_ROOT_DIR'} ||= File::Basename::dirname( File::Basename::dirname( File::Basename::dirname( Cwd::realpath($0) ) ) );
+
+my $pipeline_url = get_test_url_or_die();
+
+# Starting a first set of checks with a "GCPct" pipeline
+
+init_pipeline('Bio::EnsEMBL::Hive::Examples::FailureTest::PipeConfig::FailureTest_conf', $pipeline_url);
+
+my $hive_dba = Bio::EnsEMBL::Hive::DBSQL::DBAdaptor->new( -url => $pipeline_url );
+
+my $stdout = capture_stdout {
+    peekJob($pipeline_url, ["-job_id" => 1]);
+};
+$stdout =~ s/\s+//g;
+my $exp_stdout = "%unsubstituted_param_hash=('column_names'=>['value'],'inputlist'=>'#expr([0..#job_count#-1])expr#','job_count'=>10);";
```

Comment:
I'm concerned that this could fail if the test is run in different environments or under different Perl versions, since hash key order is random. Since this is a simple structure, you should be able to eval it to recreate the hash, then test the presence of hash keys and values.

Reply:
Yes, I noticed this and added an extra commit (7618b68) to ensure Data::Dumper sorts the keys to ensure it will always print in the same order. Can still make your suggested change if you'd prefer.

Reply:
Oops, I completely missed that. My apologies - it should be fine as is then, no need for a change.